.rustfmt.toml

```diff
@@ -8,7 +8,7 @@ fn_call_width = 60
 struct_lit_width = 18
 struct_variant_width = 35
 force_explicit_abi = true
-newline_style = "Unix"
+newline_style = "Native"
```

Comment:
Whats the motivation for changing this?

My biggest concern is people getting in the habit of working with `Native` and completely botching the tests.  It was a big weird to work through previously why someone's tests started all failing.  The person then opened #241 though I don't know if there is a good solution to it.

Reply:
For people using git with autocrlf set to true, it will avoid rustfmt to change all the file on the project :)

Reply:
The hard thing is not everyone sets that flag.  I don't when I'm on Windows.

I suspect there is no right answer here.

Reply:
Yeah :-/ or force a choice with .gitattributes as we spoke on gitter

Reply:
https://www.git-scm.com/docs/gitattributes

Reply:
So we have two options
- Keep it on `Unix` and expect people to override the file locally when using autocrlf
- Use editorconfig / gitattributes on the code as well as the tests to force unix style.

The first approach might be less than ideal because for someone being "sloppy" and doing `git -a`, they'll constantly be having to cut that change out of their commits.

Thoughts?

Reply:
I think gitattributes is safer for us

Reply:
After work on my branch https://github.com/Geobert/cobalt.rs/tree/%23241 this `Native` is not needed anymore, thanks to the `.gitattributes` file